src/Pages/StockDetail/StockDetail.tsx

```diff
@@ -1,36 +1,48 @@
-import React from 'react';
-
-import { useParams } from 'react-router-dom';
+import axios from 'axios';
+import { useParams } from 'react-router-dom'
 import { useQuery } from 'react-query';
-import { Stock } from './components/Stock/Stock';
-import { useStockUi } from './hooks/useStockUI';
+import { Stock } from '../../state/watchlist/watchlistSlice';
 
-function StockDetail() {
-  //я когда раньше использовал useParams, он мне возразал значение строкой, а сейчас приходит объект
-  const { data, loading, error } = useStockUi();
 
-  if (loading) {
-    return <h3>Loading...</h3>;
+type StockDetailUrlParams = string
+
+function StockDetail() {
+  const stock_id = useParams<StockDetailUrlParams>()
```

Comment:
В javascript в основном код обычно пишут в camelCase
```suggestion
  const stockId = useParams<StockDetailUrlParams>()
```